04fateNumber.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+* 1)	NUMBER OF FATE. The digits of the full date of birth are added together until one digit is obtained.
+*/
+
+#include <iostream>
+
+using namespace std;
+
+int input(int number, string s);
+int calculateSingleNumber(int number);
+int calculateFateNumber(int year, int month, int day);
+void printFateNumber(int fateNumber);
+void printDecription();
+
+int main()
+{
+	int year = 0, month = 0, day = 0, fateNumber;
+
+	printDecription();
+
+	year = input(year, "year: ");
+	month = input(month, "month: ");
+	day = input(day, "day: ");
+
+	//fateNumber = calculateSingleNumber(year) + calculateSingleNumber(month) + calculateSingleNumber(day);
+	//fateNumber = calculateSingleNumber(fateNumber);
+
+	fateNumber = calculateFateNumber(year, month, day);
```

Comment:
Vadim, please look at lines 25-26. It is possible to use these lines instead of line 28. In such case function `int calculateFateNumber(int year, int month, int day)` isn't required. So, one more line in main, and one function less. Could you please suggest to me what is better?

Reply:
You can use both of them. But I like `fateNumber = calculateFateNumber(year, month, day);`.
Also, you can transfer to `printFateNumber(fateNumber);` instead of `fateNumber` ---- `calculateFateNumber(year, month, day);`. I mean `printFateNumber(calculateFateNumber(year, month, day));`. It's normal.

Reply:
Thank you. Code looks more readable in such a way
![image](https://user-images.githubusercontent.com/79978383/111986543-1fe0b800-8b1f-11eb-9bd7-76b3fdf9db01.png)